bedrock/pocket/templates/pocket/home.html

```diff
@@ -0,0 +1,123 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "pocket/base.html" %}
+
+{% from "macros-protocol.html" import split, picto with context %}
+
+{% block page_title %}{{ ftl('pocket-home-homepage-title') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('pocket-home-homepage-desc') }}{% endblock %}
+
+{% block pocket_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('pocket-home') }}
+{% endblock  %}
+
+<!-- To be placed for localized images in the header split macro -->
+{# image_url='l10n_img('pocket/pocket-home-articles.png')', #}
+
+{% block content %}
+{% include 'pocket/includes/platform-nav.html' %}
+
+    <div class="pocket-homepage mzp-t-content-xl">
+        {% call split(
+          block_class='pocket-homepage-header mzp-l-split-pop',
+          media_class='mzp-l-split-media-overflow',
+          media_after=True,
```

Comment:
There's an l10n param we should be able to use here (but that might change depending on how we decide to do localized Pocket images). This should work for now :) 
```
          l10n_image=True,
          image_url='pocket/pocket-home-articles.png',
```